partitioner/src/main/java/io/confluent/connect/storage/partitioner/TimeBasedPartitioner.java

```diff
@@ -100,10 +123,16 @@ public String encodePartition(SinkRecord sinkRecord) {
     Class<? extends SchemaGenerator<T>> generatorClass = null;
     try {
       generatorClass =
-          (Class<? extends SchemaGenerator<T>>) config.get(PartitionerConfig.SCHEMA_GENERATOR_CLASS_CONFIG);
+          (Class<? extends SchemaGenerator<T>>) config.get(
+              PartitionerConfig.SCHEMA_GENERATOR_CLASS_CONFIG
+          );
       return generatorClass.getConstructor(Map.class).newInstance(config);
-    } catch (ClassCastException | IllegalAccessException | InstantiationException | InvocationTargetException
-        | NoSuchMethodException e) {
+    } catch (ClassCastException
```

Comment:
Not sure about this one. Again not a method call. And `ClassCastException` is not on its own line and the indentation of the final `) {` is not exactly at the right level probably. 

Checkstyle is complaining at `OperatorWrap` but not if I write `catch` as follows: 

```java
tor(confClass, String.class);
      return constructor.newInstance(conf, url);
    } catch (NoSuchMethodException
        | InvocationTargetException
        | InstantiationException
        | IllegalAccessException e) {
      throw new ConnectException(e);
    }
```
it's not throwing an error. Your call.